src/test/FuzzerImpl.cpp

```diff
@@ -1091,122 +1091,122 @@ std::array<
            AssetID(1), 0, DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
            DEFAULT_SPARE_TRUST_LINE_LIMIT,
            {ClaimableBalanceParameters(DEFAULT_CLAIMABLE_BALANCE_AMOUNT),
-            ClaimableBalanceParameters(0)}),
+            ClaimableBalanceParameters()}),
```

Comment:
Now it's obvious that nothing actually has 2 claimable balances. Why make the array length 2 if you're going to only use 1?

Reply:
I had planned on using more in the future "policy" commits; I was trying to avoid changing more than necessary in the earlier ones.  In light of your argument for separating `ClaimableBalanceParameters` out of `TrustLineParameters` and into their own array, this will be resolved automatically (we won't need to pick some specific number of claimable balances per trust line).